realm/src/main/java/io/realm/Realm.java

```diff
@@ -1403,6 +1694,73 @@ public void executeTransaction(Transaction transaction) {
     }
 
     /**
+     * Similar to {@link #executeTransaction(Transaction)} but runs asynchronously from a worker thread
+     * @param transaction {@link io.realm.Realm.Transaction} to execute.
+     * @param callback optional, to receive the result of this query
+     * @return A {@link io.realm.Realm.Request} representing a cancellable task
+     */
+    public Request executeTransaction(final Transaction transaction, final Transaction.Callback callback) {
```

Comment:
Because I'm a big fan of Retofit, I'd prefer `executeTransaction(Transaction, Callback)`.